pkgs/applications/version-management/gitoxide/default.nix

```
@@ -1,24 +1,29 @@
-{ lib, rustPlatform, fetchFromGitHub }:
+{ lib, rustPlatform, fetchFromGitHub, openssl, pkgconfig }:
 
 rustPlatform.buildRustPackage rec {
   pname = "gitoxide";
-  version = "0.3.0";
+  version = "0.4.0";
 
   src = fetchFromGitHub {
     owner = "Byron";
     repo = "gitoxide";
     rev = "v${version}";
-    sha256 = "0xpic9jx7nrxi5d8lqch2vxpvipx994d717c4n0kgr3ipyij1347";
+    sha256 = "0sx3z9l9n9qq2zj91pgm7znhxjsj59zvwms3aivfglhawwj3iwyj";
   };
 
-  cargoSha256 = "104lyfni75h1i30s2jlzf66sp1czfd9ywqz78kj4i7lfdf6fc4x9";
+  cargoSha256 = "0ykkh86p4csi0v3pb2idjk94w9m32a34a5qrvna7ml5yz84m8hva";
+
+  nativeBuildInputs = [ pkgconfig ];
+  buildInputs = [ openssl ];
+
+  # Needed to get openssl-sys to use pkgconfig.
+  OPENSSL_NO_VENDOR = 1;
 
   meta = with lib; {
     description =
       "A command-line application for interacting with git repositories";
     homepage = "https://github.com/Byron/gitoxide";
-    # NOTE: the master branch is dual-licensed with APACHE but v0.3.0 is only MIT
-    license = licenses.mit;
+    license = with licenses; [ mit asl20 ];
```

```suggestion
    license = with licenses; [ mit /* or */ asl20 ];
```

Which it the convention if either license can be used.